bedrock/firefox/redirects.py

```diff
@@ -58,80 +56,80 @@ def mobile_app(request, *args, **kwargs):
 
 # Issue 16355
 springfield_redirectpatterns = (
-    offsite_redirect(r"^firefox/$", f"{FXC}/", permanent=PERM_REDIRECTS),
-    offsite_redirect(r"^firefox/browsers/$", f"{FXC}/", permanent=PERM_REDIRECTS),
-    offsite_redirect(r"^firefox/new/$", f"{FXC}/", permanent=PERM_REDIRECTS),
+    offsite_redirect(r"^firefox/$", f"{FXC}/", permanent=True),
```

Comment:
Nit, non-blocking: By default I think the `redirect` helper, which is called by `offsite_redirect`, defaults to `permanent=True` anyway. However this is making it explicit and easier to learn from so I don't mind keeping it like this.